### PR TITLE
Upgrade mpfr to the latest upstream release 4

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@isuruf, so once https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/65 has been merged I can drop the pin on gmp and mpfr, right?

**Comment by isuruf** [comment]
Yes

**Comment by isuruf** [comment]
You have to rerender though

**Comment by saraedum** [comment]
Yes, you are right.

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,18 +10,21 @@ source:
...
   skip: true        # [win]
+  run_exports:
+    - {{ pin_subpackage('libflint', max_pin='x.x') }}
 
 requirements:
   build:
-    - toolchain
-    - gmp >=5.0.1,<7
-    - mpfr 3.*
+    - {{ compiler('c') }}
```
[the remark this answers is not in the document]
Comment (saraedum) [reply recipe/meta.yaml]: Does cxx stand for c++?

[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/meta.yaml]: Yes. Flint has C++ bindings.

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Pull request merged**